### PR TITLE
Generate API list

### PR DESCRIPTION
**Pull request by randres1640**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,4 +49,18 @@ export RACK_ENV=test
...
 rake tests:api
-```
\ No newline at end of file
+```
+
+## List APIs
+
+API routes should be annotated this way
+
+```shell
```
Comment (doggygit) [inline README.md:58]:
```suggestion
```ruby
 # route[List_wards]: /api/wards 
 r.get do
 ...
 end
```

File: Rakefile [hunk 2]
```diff
@@ -59,3 +59,10 @@ namespace :tests do
     spec.call('./spec/api/*_spec.rb')
   end
 end
+namespace :assets do
+  desc "Update the routes metadata"
+  task :precompile do
+    sh 'grep -rh "# route" api > routes.tmp'
+    sh 'roda-parse_routes -f routes.json routes.tmp'
```
Comment (doggygit) [inline Rakefile:66]: If we would add a `jq . < routes.json` we get something thats easier to read for humans :)

Comment (randres1640) [reply Rakefile:66]: $ jq
sh.exe": jq: command not found

I don't like humans, except jeremyevans, of course, XD.

How does jq . looks like? I can make a ruby oneliner with the same output

Comment (doggygit) [reply Rakefile:66]: > I don't like humans, except jeremyevans, of course, XD.

:laughing: 

jq is only available when it is installed of course. It personally my favorite tool to quickly view json files in the terminal.

But I don't insist on it :)

**Pull request merged**